src/services/pathCompletions.ts

```diff
@@ -27,7 +27,7 @@ namespace ts.Completions.PathCompletions {
         const scriptDirectory = getDirectoryPath(scriptPath);
 
         if (isPathRelativeToScript(literalValue) || isRootedDiskPath(literalValue)) {
-            const extensions = getSupportedExtensions(compilerOptions);
+            const extensions = getSupportExtensionsForModuleResolution(compilerOptions);

```

Comment:
*supported

Reply:
Shouldn't we just change `getSupportedExtensions`? That's used by `isSupportedSourceFileName` which is used by `watch.ts`, and presumably we want to watch for changes to '.json' files if they're included in the program?

Reply:
```isSupportedSourceFileName``` which internally uses ```getSupportedExtensions``` is used to see if the file name is valid when getting the fileNames to compile. We do not add json files then and we shouldnt because we only want to add them through module resolution.